.github/workflows/guru-reviewer.yml

```diff
@@ -0,0 +1,42 @@
+name: Analyze with CodeGuru Reviewer
+
+on: [push]
+
+permissions:
+    id-token: write
+    contents: write
+    security-events: write 
+
+jobs:
+  build:
+    name: Clone your repository
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+        submodules: 'true'
+    - name: Set up JDK 1.8
+      uses: actions/setup-java@v1
+      with:
+        java-version: 1.8
+    - name: Build with Maven
+      run: mvn compile -DskipTests
+
+    - name: Configure AWS credentials from Test account
+      uses: aws-actions/configure-aws-credentials@master
+      with:
+        role-to-assume: arn:aws:iam::048169001733:role/GuruGitHubCICDRole
```

Comment:
As discussed, this is an internal account from CodeGuru Reviewer to be used for other repositories in the org. We should be able to see the results in GitHub in the Security tab. 

Thanks a lot for this PR, Martin!